tensorflow/docs_src/install/install_sources.md

```diff
@@ -429,3 +429,41 @@ Stack Overflow and specify the `tensorflow` tag.
   <pre>ImportError: cannot import name pywrap_tensorflow</pre></td>
 </tr>
 </table>
+
+## Supported source configurations
```

Comment:
Let's say "Tested"
we may be supporting more than what is listed here.

Reply:
Done.